redux-api/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@shoutem/redux-api",
+  "version": "0.1.1-beta.7",
+  "description": "Redux SDK for Shoutem API",
+  "main": "build/lib/index.js",
+  "files": [
+    "build/lib"
+  ],
+  "author": "Shoutem",
+  "scripts": {
+    "clean": "rimraf ./build/*",
+    "build": "npm run clean && npm run build-lib",
+    "build-lib": "babel src --out-dir build/lib --copy-files",
+    "test": "mocha --require babelTestSetup --reporter spec --recursive test",
+    "coverage": "babel-node node_modules/isparta/bin/isparta cover --report text --report html node_modules/mocha/bin/_mocha -- -R spec --recursive test",
+    "lint": "eslint src test"
+  },
+  "devDependencies": {
+    "babel-cli": "^6.9.0",
+    "babel-core": "^6.9.1",
+    "babel-eslint": "^6.0.0",
+    "babel-preset-es2015": "^6.9.0",
+    "babel-preset-stage-0": "^6.3.13",
+    "babel-register": "^6.9.0",
+    "chai": "^3.5.0",
+    "chai-shallow-deep-equal": "^1.4.0",
+    "deep-freeze": "0.0.1",
+    "isparta": "^4.0.0",
+    "istanbul": "0.4.4",
+    "mocha": "^2.5.3",
+    "nock": "^8.2.1",
+    "rimraf": "^2.5.2",
+    "sinon": "^1.17.4"
+  },
+  "dependencies": {
+    "@shoutem/js-api": "file:../fetch-api",
+    "@shoutem/js-api": "0.1.1-beta.2",
```

Comment:
Yes, this is map, and last key overrides all the rest keys. If you want to test fetch-api in redux-api link allows you to do it quickly. Json doesnt support comments :(